src/dev/getting_started.md

```diff
@@ -0,0 +1,100 @@
+# Building and Running Tests
+
+## Repos
+
+Yorick is spread over a handful of repos, but the two you are likely to need to clone are:
+
+ - [ykrustc](https://github.com/softdevteam/ykrustc): The compiler.
+ - [yk](https://github.com/softdevteam/ykrustc): The runtime parts of the system.
+
+The latter is a monorepo containing a few different crates.
+
+## Building and Testing the Compiler
+
+### Building the Compiler
+
+To build the compiler, run:
+```
+./x.py build --stage 1
+```
+
+A stage one build is usually sufficient for development.
+
+If you have made changes to `ykpack` (in the `yk` repo), then edit the
```

Comment:
I'm not sure if this should *really* be in the `Cargo.toml`: I'd have been better off not uncommenting this at first, I think. It feels like something that developers should know to do, and now it's documented here, it shouldn't be in the `Cargo.toml`?

Reply:
It is in `Cargo.toml`. Do you mean it should be in the `Cargo.toml` of  crates, rather than at the top-level?

The problem is:
 - There are multiple places (currently two) where you'd have to override the dep and fiddling around finding them is a pain.
 - During development we change the override quite frequently, especially if we are doing 3-stage PRs.

Being able to quickly uncomment these lines in one easily locatable place is a real boon, and I wouldn't like to change it.

Reply:
The problem with this is that we're putting in something for our developer convenience that will confuse everyone else. It's already confused me. It's basically a debug `println` and you know how much I like those ;)

Reply:
I feel pretty strongly that I'd like to keep this. I don't enjoy re-typing those lines.

@ptersilie your opinion on this?

Reply:
I agree with Edd, that having this in here is really helpful. Though I also get your reasoning. Maybe there's a solution that fits all?

Reply:
I've just found `x.py --config <file>`, but it's still kind of annoying, as it's yet another thing to remember.

Thoughts?

Reply:
If it's documented on the wiki, and we copy and paste it (when necessary) into our local copy, I'm not sure I see a real problem.

Reply:
The problem is, we can't escape editing `Cargo.toml` as it's needed in stage 4 of [breaking cycles](https://softdevteam.github.io/ykdocs/misc/ci-cycles.html), so if we introduce a local copy of the config file, we'd have two config files now instead of one.